.changes/unreleased/tinymce-TINY-12092-2025-05-13.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: DomPaser not removing whitespace around `colgroup` and `col` elements.
```

Whitepace might be the reason for this, but anyone who cares about the schema will want to know the actual detail
```suggestion
body: HTML schema did not treat `colgroup` or `col` elements as block tags.
```

---

Maybe use `HTML schema did not treat `colgroup` or `col` as block elements.`